lib/web_ui/lib/src/engine/text_editing/input_type.dart

```diff
@@ -66,7 +66,7 @@ abstract class EngineInputType {
   html.HtmlElement createDomElement() => html.InputElement();
 
   /// Given a [domElement], set attributes that are specific to this input type.
-  void configureInputMode(html.HtmlElement domElement) {
+  void configureInputMode(html.HtmlElement? domElement) {
```

Comment:
When calling this method, `dom_element` should not be null.

Reply:
Done